service/history/transferQueueActiveProcessor.go

```diff
@@ -21,7 +21,7 @@
 package history
 
 import (
-	"context"
+	ctx "context"
```

Comment:
I think it has little benefits to shortcut context. 
People get used to use `context` package already and when people read ctx.Func they may have question mark..

Reply:
the thing is, most context used by history service is workflow execution context...
we can determine whether make that ctx and go's context a context

Reply:
my preference is using context for standard go package, and name workflow context explicit wfCtx, so that whoever read code will have no suprise